src/prefect/server/models/deployments.py

```diff
@@ -619,6 +619,7 @@ async def _generate_scheduled_flow_runs(
                     "id": uuid4(),
                     "flow_id": deployment.flow_id,
                     "deployment_id": deployment_id,
+                    "deplyment_version": deployment.version,
```

Comment:
deplyment_version is this an issue?

Reply:
@meggers 
> deplyment_version is this an issue?

It shouldn't be no, but for clarification are you just referring to the making convention?

Reply:
Sorry for my lack of clarity, was referring to the misspelling (missing o in deployment)

Reply:
Ah yep thanks for flagging that I'll get it updated 😁